packages/building-utils/index-html/polyfills.js

```diff
@@ -156,8 +156,7 @@ function getPolyfills(config) {
       instructions.push({
         name: 'es-module-shims',
         test: "'noModule' in HTMLScriptElement.prototype",
-        path: require.resolve('es-module-shims/dist/es-module-shims.min.js'),
-        sourcemapPath: require.resolve('es-module-shims/dist/es-module-shims.min.js.map'),
+        path: require.resolve('es-module-shims/dist/es-module-shims.js'),
```

Comment:
There is no more minified output in `es-module-shims` 1.x and the structure looks like this:

![Screenshot 2022-11-19 at 21 25 52](https://user-images.githubusercontent.com/10589913/202868013-54154474-a080-4cb8-980c-dea911a9f8bf.png)